svy/src/helpers/protocol.ts

```diff
@@ -56,3 +69,60 @@ export function decrementVeSVYHolder(block: ethereum.Block): void {
   protocol.save();
   createProtocolSnapshot(block, protocol);
 }
+
+export function getCirculatingSVY(block: ethereum.Block): BigInt {
+  const timestamp = block.timestamp;
+  let totalLockedSVY = BIGINT_ZERO;
+
+  /**
+   * DEV: Closures (functions with a captured environment) are not yet supported.
+   * Learn more: https://www.assemblyscript.org/status.html#on-closures
+   */
+
+  const streamingHedgeysTokenIds = createBigIntArray(28, 29);
+  for (let index = 0; index < streamingHedgeysTokenIds.length; index++) {
+    totalLockedSVY = totalLockedSVY.plus(
+      streamingHedgeysContract.streamBalanceOf(streamingHedgeysTokenIds[index]).getRemainder()
+    );
+  };
+
+  const tokenLockupPlansPlanIds = createBigIntArray(5, 81);
```

Comment:
These can be stored in constants wdyt?

Reply:
In case these range values are stored in constants file, it can become verbose, imo.
I renamed `tokenLockupPlansPlanIds` and `tokenVestingPlansPlanIds` to `tokenLockupPlanIds` and `tokenVestingPlanIds`.